imports/plugins/core/checkout/client/templates/checkout/completed/completed.html

```diff
@@ -35,13 +35,7 @@
     </div>
   {{else}}
     <div class="container">
-      <div class="row text-center">
-        <h1>Order Processing</h1>
-        <p>Please do not click back button.</p>
-        <br />
-        <br />
-        <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
-      </div>
+      {{> unauthorized}}
```

Comment:
So this change throws "unauthorized" whenever the order is not completed. What if the user is valid but the order is not completed (which can happen with something like Paypal which can take a few seconds)? Rather than eliminating the "Order Processing" page, you need to add another check for whether that user is the user for this order.